test/syscall/syscall.py

```diff
@@ -968,11 +968,15 @@ def execute(self):
 wt_disttop = os.path.dirname(os.path.dirname(syscalldir))
 
 # Note: this code is borrowed from test/suite/run.py
-# Check for a local build that contains the wt utility. First check in
-# current working directory, then in build_posix and finally in the disttop
-# directory. This isn't ideal - if a user has multiple builds in a tree we
-# could pick the wrong one.
-if os.path.isfile(os.path.join(os.getcwd(), 'wt')):
+# Check for a local build that contains the wt utility. First check if the user
+# manually specified a local build through the 'WT_BUILDDIR' env variable. Otherwise
+# iterate through other possible locations. This including current working directory,
+# then in build_posix and finally in the disttop directory. This isn't ideal - if a
```

Comment:
Typo `disttop`

Reply:
I believe the original comment I adapted this from was referring to the variable `disttop`. Anyways I renamed it to just `dist` 47c1a56.